modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-ack.adoc

```diff
@@ -0,0 +1,153 @@
+= Anypoint MQ ACK and NACK Operations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+== Acknowledgment Configurations
+
+The Anypoint MQ connector provides three acknowledgment configurations: Immediate, Automatic, and Manual.
+
+=== Immediate Acknowledgment
+
+To acknowledge messages as your app consumes them, before processing them, set the `acknowledgementMode` to `IMMEDIATE` either in the Subscriber or Consume operations.
+
+Acknowledging the message automatically when it's received means that the message isn't redelivered if an error occurs during message processing. To handle errors without losing messages, use a dead letter queue.
+
+If an error occurs while trying to acknowledge the message using `IMMEDIATE` mode, an `ANYPOINT-MQ:ACKING` error is thrown.
+
+=== Automatic Acknowledgment
+
+To acknowledge a received message automatically only if the flow execution completes successfully, set the `acknowledgementMode` to `AUTO` in the Subscriber operation.
+
+If an error occurs during the flow execution, causing its execution to terminate prematurely, the message is NACKed and returns to the queue for redelivery.
+
+=== Manual Acknowledgment
+
+The `MANUAL` acknowledgment mode delegates all responsibility to the app logic to decide when to acknowledge the message, using the ACK or NACK operations.
+
+With this configuration, every message received by either the Subscriber or Consume operations has an `ackToken` value available in the message attributes that identifies this message uniquely for a given connection.
```

Comment:
Here and earlier, I suggest "...by either the Subscriber or Consume operations..."  
>  "...by either the Subscriber operation or the Consume operation...."
I think it just sounds better because "subscriber" and "consume" are not parallel.

Do we need "available in the message attributes"?  "Every message...has an 'ackToken' value that identifies...."